src/script/lua_api/l_object.cpp

```diff
@@ -1720,6 +1720,45 @@ int ObjectRef::l_get_day_night_ratio(lua_State *L)
 	return 1;
 }
 
+// set_time_offset(self, offset)
+int ObjectRef::l_set_time_offset(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	RemotePlayer *player = getplayer(ref);
+	if (player == NULL)
```

Comment:
`!player`

Reply:
Every other function in this file is with player == NULL too.

Reply:
The fact that other functions do the same doesn't mean it's good code.

Reply:
That was my first real C++-Code so I was inspired by the other code.